src_blog/_posts/New-Release-Grain-v0.5-Durum.md

```diff
@@ -0,0 +1,97 @@
+
+With some tweaks to our code generation and how we statically link compiled modules together, we’re now able to produce binaries that are a fair bit smaller (30% smaller for hello world) and in some cases, significantly faster. Although all Grain functions are first-class values, in many cases we’re able to tell exactly how they’re used and emit faster function call instructions.
+
+## Developer Experience
+
+We care deeply about developer experience and make sure we’re improving it with every single release. This time is certainly no exception and just the start of what’s to come in future releases.
+
+### Faster binaries
+
+We’ve heard the community loud and clear—the pre-built binaries were painfully slow. To that end, the team spent this release cycle working tirelessly to speed up various slow paths in the compiler. With these improvements, compiling a program is an order of magnitude faster.
+
+We’ll talk about the [new parser](#new-parser) and [compilation profiles](#compilation-profiles) below, which were the biggest improvements by far.
+
+If you build Grain from source, we now fully support the Mac M1 architecture. Both the initial compilation and incremental builds are much, much faster on an M1 computer. In the near future, we hope to provide arm64 Docker images, but that will take some more work.
+
+### New parser
+
+This release ships with a brand new parser. Not only is this parser significantly faster than the old parser, but it also brings nearly 200 unique error messages for when a syntax error occurs—providing you with specific details about what went wrong and how you can fix it.
+
+![grain parser error](/blog/photos/misc/grain-parser-error.jpg)
+
+### Compilation profiles
+
+To make the configuration for building modules for production much easier, we’ve introduced compilation profiles. Initially, we have just one—`release`—which enables all of the best-in-class WebAssembly optimizations available today. After some profiling, we discovered that running these optimizations accounted for more than 50% of compilation time, and they’re really only necessary when you’re ready to ship a release of your module. The release profile can be enabled via the `--release` flag, and you can learn more about it in [the corresponding documentation](https://grain-lang.org/docs/tooling/building_for_production).
+
+### New LSP
+
+Previously, Grain editor support was exclusive to Visual Studio Code due to a required shim layer between the extension and compiler. In this release of Grain, core team member [Marcus](https://twitter.com/marcusr) added a long-running language server directly into the `grain` command that speaks the language server protocol.
+
+This means that the newest version of the VSCode extension is exclusively a “language client” that launches `grain lsp`, and any other editor that supports the language server protocol will be able to integrate seamlessly. Let us know what editors you want to see support for!
```

Comment:
We could add that Emacs support is almost done, or if I hurry up, say it is done.  Also the new LSP has a huge performance increase over the previous version, and produces higher quality results for hovers